codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/ActionServiceSourceGenerator.scala

```diff
@@ -56,16 +55,16 @@ object ActionServiceSourceGenerator {
     val interfaceSourcePath =
       generatedSourceDirectory.resolve(packagePath.resolve(interfaceClassName + ".java"))
 
-    val _ = interfaceSourcePath.getParent.toFile.mkdirs()
-    val _ = Files.write(
+    interfaceSourcePath.getParent.toFile.mkdirs()
+    Files.write(
```

Comment:
I may be wrong, but I remember to have used `val _ = ` more then once in a Scala 2.12 build. 

It turns out that the second `_` fails because `_` is already defined. 🤷 

Anyway, since the we are not enforcing it during compilation, I thought it would just simpler to not capture the return.